package.json

```diff
@@ -40,24 +46,22 @@
   },
   "homepage": "https://github.com/Asana/tsutil",
   "devDependencies": {
-    "chai": "^2.1.0",
-    "del": "^1.1.1",
+    "chai": "^2.3.0",
+    "coveralls": "^2.11.2",
     "dts-bundle": "^0.2.0",
-    "event-stream": "^3.2.2",
-    "glob": "^4.4.0",
-    "gulp": "^3.8.11",
-    "gulp-gh-pages": "^0.4.0",
-    "gulp-istanbul": "^0.5.0",
-    "gulp-mocha": "^2.0.0",
-    "gulp-replace": "^0.5.3",
-    "gulp-tslint": "^1.4.3",
-    "gulp-typedoc": "^1.0.6",
-    "gulp-typescript": "^2.4.2",
-    "sinon": "^1.12.2",
-    "typescript-formatter": "^0.1.4",
-    "typescript-require": "pspeter3/typescript-require"
+    "istanbul": "^0.3.13",
```

Comment:
Yeah, that was the most exciting part. It also seems like it runs faster too now.